### DISCUSSION TITLE
trustservercertificate=true possible

### DISCUSSION THREAD

**Discussion by tovoro** [lowlydba/lowlydba.sqlserver#212] [General]
Is it possible to add connection strings like the above somewhere? 

I get a "The certificate chain was issued by an authority that is not trusted" error while trying to backup / restore a db to a development mssql server and would like to circumvent that with the connection parameter.

**Comment by lowlydba** [comment]
I recommend running a variation of [Set-DbatoolsInsecureConnection](https://docs.dbatools.io/Set-DbatoolsInsecureConnection.html) on an as-needed basis to work around this. DBATools handles this as a setting config, so it isn't really necessary to add it to every connection string.

You can see we use this approach during our initial CI setup to fix the same problem: https://github.com/lowlydba/lowlydba.sqlserver/blob/bee0380751c2fa0b26ec381fc4f9a166fcb33e85/tests/integration/targets/setup_sqlserver/vars/main.yml#L5-L6
